### ISSUE TITLE
AccountEvents vs Events in accountHolderStatus

### ISSUE THREAD

**Issue by andras-knuth** [Adyen/adyen-java-api-library#521]
Hey! So my company is at the beginning of the process to integrate adyen payment and we face a possible issue. 
In the api docs the accountHolderStatus has an array of accountEvents called **events** - meanwhile the official library (v 13.0, but it is the same in 13.1), what we also use in our application has a list of AccountEvents called **accountEvents**. Because of this, we have an empty array (while calling the test service serves us with the correct data). We would like to use the official library to save effort, but as things see now it is not working. 
To summarize - the naming of events on the getAccountHolder call is different in the api and the library. 
![adyen accountHolderStatus](https://user-images.githubusercontent.com/82878590/115393977-96d2a480-a1e2-11eb-8abc-a6211574f599.png)
![adyenAccount'HolderStatus-api](https://user-images.githubusercontent.com/82878590/115393985-9934fe80-a1e2-11eb-8309-3ab3ba54c909.png)

**Comment by Aleffio** [comment]
Hi @andras-knuth ,
Thanks for reporting the issue,
We will push a fix for it, it will be available in the next release.

Kind Regards,
Alessio
Adyen

**Comment by andras-knuth** [comment]
Well guys, that was fast, thanks!
